pkg/config/config.go

```diff
@@ -96,8 +96,8 @@ func init() {
 	Datadog.SetDefault("GUI_port", defaultGuiPort)
 	if IsContainerized() {
 		Datadog.SetDefault("container_proc_root", "/host/proc")
+		Datadog.SetDefault("procfs_path", "/host/proc")
```

Comment:
`/host/proc` needs to be the default for the docker image but not for the whole agent, you can remove that default and only add the envvar binding

Reply:
Isn't **IsContainerized** designed for that ? Should I move the envvar binding in this if block ?

Reply:
indeed, it this case we don't need to add it to `DD_PROCFS_PATH`